.travis.yml

```diff
@@ -1,9 +1,12 @@
 ---
 git:
   depth: 10
-language: generic
+language: node_js
 
-sudo: false
+node_js:
+  - "6"
+
+sudo: required # Until Yarn repo is added to apt-source-whitelist
 
 addons:
```

Comment:
The whitelist seems to take a long time to get things into it, there's [225 issues and 40 pull requests](https://github.com/travis-ci/apt-source-whitelist) right now.

CircleCI is much faster than Travis and allows `sudo` by default, so we're mostly relying on Circle these days instead of Travis. Travis' container infra is more limited than Circle's, Circle even lets you SSH into the build boxes to diagnose build problems.